agents/craftassist/craftassist_agent.py

```diff
@@ -87,6 +87,14 @@ def __init__(self, opts):
         ]
         self.perceive_on_chat = True
 
+        # Add optional logging for timeline
+        if opts.log_timeline:
+            timeline_log = open("timeline_log.txt", "w")
```

Comment:
You should add some kind of id to the log file. Please refer to how to name agent log file: https://github.com/facebookresearch/droidlet/blob/amanda/timeline/agents/craftassist/craftassist_agent.py#L145